src/styles/palette.js

```diff
@@ -64,6 +65,18 @@ export default function createPalette({
   error = red,
   type = 'light',
 } = {}) {
+  if (difference(keys(indigo), keys(primary)).length) {
+    throw new Error('Invalid primary color');
```

Comment:
I can enumerate the expected attributes in the message and refer them to the default color.